StarVMC/StarGeometry/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.h
+*.cxx
```

Comment:
Let's see if #148 works. If it does we won't be polluting the checked out source with generated files. In any event, we should modify such files in a separate PR outside of this one